package.json

```diff
@@ -32,24 +32,24 @@
   },
   "devDependencies": {
     "@types/node": "^8.0.47",
+    "airtap": "^0.0.1",
     "faucet": "^0.0.1",
     "istanbul": "^0.4.2",
     "istanbul-coveralls": "^1.0.3",
     "rimraf": "^2.6.2",
     "standard": "^10.0.3",
     "tape": "^4.8.0",
     "ts-node": "^4.1.0",
-    "typescript": "^2.6.2",
-    "zuul": "github:vweevers/zuul#custom-loopback-hostname"
+    "typescript": "^2.6.2"
   },
   "browser": {
     "./immediate.js": "./immediate-browser.js"
   },
   "scripts": {
     "test": "standard && node test.js --stderr | faucet && npm run test-ts",
     "test-ts": "ts-node --no-cache test.js | faucet",
-    "test-browsers": "zuul --browser-retries 2 --sauce-connect --concurrency 1 --loopback zuul.local --no-coverage ./test.js",
-    "test-browser-local": "zuul --ui tape --no-coverage --local 9000 ./test.js",
+    "test-browsers": "airtap --sauce-connect --loopback airtap.local --no-coverage test.js",
```

Comment:
Note that the default `--browser-retries` is 6, which can delay the build somewhat, but helps against random startup failures (over at Sauce Labs).

The default `--concurrency` is 5, which (atm) is working well in `airtap` itself so I also want to try it here.